plugins/modules/ec2_vpc_nat_gateway.py

```diff
@@ -75,6 +75,11 @@
         When specifying this option, ensure you specify the eip_address parameter
         as well otherwise any subsequent runs will fail.
     type: str
+  default_create:
+    description:
+      - if EIP address is not found, create NAT gateway with EIP address as null.
```

Comment:
I guess the description of this parameter must be carified. What happens when the users specifies an invalid allocation_id? Does default_create: True allow to still create the NAT gateway and allocate a new EIP?

Something like this:
- When I(default_create=True), if I(eip_address) is set and it is invalid or does not exist, the NAT gateway is successfully created and a new EIP is automatically allocated.
-  When I(default_create=True), if I(eip_address) is set and it is invalid or not found, the NAT gateway is successfully created and a new EIP is automatically allocated. (If this is true!!!)
- When I(default_create=False), what happens?

Please feel free to rephrase and correct my suggestions.